opencti-platform/opencti-graphql/src/utils/safeEjs.client.ts

```diff
@@ -0,0 +1,157 @@
+import { Worker } from 'worker_threads';
+import * as path from 'path';
+import type { Data } from 'ejs';
+import type { SafeRenderOptions } from './safeEjs';
+import type { WorkerReply } from './safeEjs.worker';
+
+export type WorkerOptions = {
+  timeout?: number,
+  useJsonEscape?: boolean,
+  resourceLimits?: {
+    maxOldGenerationSizeMb?: number
+    maxYoungGenerationSizeMb: number,
+    codeRangeSizeMb: number,
+    stackSizeMb: number,
+  }
+};
+
+// Helper to serialize functions and validate data
+const prepareDataForWorker = (data: Data): Data => {
+  const forbidden = ['eval', 'Function', 'constructor', '__proto__', 'prototype', 'freeze'];
```

Comment:
how can it happen to have such forbidden name since everything is controlled in code now ?